examples/k8s/unifi_poller.yaml

```diff
@@ -0,0 +1,82 @@
+#Create Unifi Poller
+--- 
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: unifi-poller
+  namespace: monitoring
+  labels:
+    app: unifi-poller
+    type: poller
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: unifi-poller
+      type: poller
+  template:
+    metadata:
+      labels:
+        app: unifi-poller
+        type: poller
+      annotations:
+        prometheus.io/path: /metrics
+        prometheus.io/port: "9130"
+        prometheus.io/scrape: "true"
+    spec:
+      containers:
+      - name: unifi-poller
+        image: golift/unifi-poller:latest
+        ports:
+        - containerPort: 9130
+          name: tcp
+          protocol: TCP
+        - containerPort: 9130
+          name: udp
+          protocol: UDP
+        volumeMounts:
+        - name: config-volume
+          mountPath: /config/unifi-poller.conf
+          subPath: unifi-poller.conf
+      volumes:
+      - name: config-volume
+        configMap:
+          name: unifi-poller
+
+
+# Add Config Map for Unifi Poller
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: unifi-poller
+  namespace: monitoring
+  labels:
+    app: unifi-poller
+    type: poller
+data:
+  unifi-poller.conf: |
```

Comment:
Is this easier than using env variables?

Reply:
I personally prefer to use a config map. This helps a lot when using multiple sites ... makes config a little more readable because the env variables are part of the deployment yaml and bloats it a bit up. It would be also possible to use Kubernetes secrets for passwords. At a later point i might refine this. As a initial help and version this is more than enough to get us going here.